api/app/serializers/user_serializer.rb

```diff
@@ -6,13 +6,13 @@ class UserSerializer < ActiveModel::Serializer
   attributes :id, :email, :nickname, :first_name, :last_name, :role, :created_at,
              :updated_at, :full_name, :avatar_styles, :is_current_user, :persistent_ui
 
+  has_many :favorites, serializer: FavoriteSerializer
```

Comment:
Why add this back in? Didn't you remove the favorites relation on users in an earlier commit? I don't think we need to serializer all a user's favorites unless that user is the current_user (which comes from the other serializer)